Cargo.toml

```diff
@@ -151,6 +151,7 @@ urlencoding = { version = "2.1.3" }
 walkdir = { version = "2.5.0" }
 which = { version = "6.0.0" }
 winapi = { version = "0.3.9", features = ["fileapi", "handleapi", "ioapiset", "winbase", "winioctl", "winnt"] }
+winreg = { version = "0.52.0" }
```

Comment:
Already in our dependency tree. I'd slightly rather use `winapi` if possible but `rustup` uses `winreg` and it's just a pain to translate it.